### DISCUSSION TITLE
How to use Owl carousel with next js

### DISCUSSION THREAD

**Discussion by Akshaypal8** [vercel/next.js#15427] [Help]
I want to use Owl carousel with my next + react js app but getting issues.

https://www.npmjs.com/package/react-owl-carousel want to use this carousel

**Comment by jamesmosier** [comment]
hi @Akki8484. Do you have a repository that you could share with what you have so far?

**Reply by Haseeb-Pervaiz** [reply]
[https://www.npmjs.com/package/react-owl-carousel]

@jamesmosier  please help

**Reply by jamesmosier** [reply]
@Haseeb-Pervaiz Do you have a repository that you could share with what you have so far?
